README.md

```diff
@@ -56,11 +56,25 @@ _Note_: To contribute, send a pull request to this repository. Note that this re
 ### Development Platform
 
 - [ONNX Runtime](https://onnxruntime.ai/): Platform agnostic model runtime to use ML models.
+- [TensorFlow](https://www.tensorflow.org/): An end-to-end open source platform for machine learning.
```

Comment:
Maybe we could add TF and torch way below under this section under a part called Frameworks. but ideally within the scope of this repository, these two should be used for their pretrained models

Reply:
Both of them have hub (PyTorch hub/Kaggle(tf hub they're moving to Kaggle) I can move under framework and those hubs for pointing that "you can quickly use it" no ML dev or train just use it

Reply:
that'd be great